### ISSUE TITLE
Go To Definition stopped working for Types

### ISSUE THREAD

**Issue by bensussman** [joefitzgerald/go-plus#824]
### Prerequisites

* [x] Have you tried launching `atom .` from the terminal in your project's directory?
* [x] Have you verified the output from `go env` is correct? If it is, please include the output in this issue.
* [x] Have you updated Atom to the latest version?
* [ ] Have you tried using [Atom Beta](https://atom.io/beta), which can be run side-by-side with Atom Stable? Is the behavior the same, or different?
* [x] Have you updated your Atom packages to the latest versions?
* [x] Have you read the [FAQ](https://github.com/joefitzgerald/go-plus/wiki/FAQ)?
* [x] Have you searched [the issues](https://github.com/joefitzgerald/go-plus/issues?utf8=%E2%9C%93&q=is%3Aissue) to see if others are experiencing the same issue?

### Description

Hyperclick "go to definition" no longer works on types for newest version of go-plus, hyperclick, atom.

### Output from `atom -v && apm -v`

```
Atom    : 1.32.1
Electron: 2.0.9
Chrome  : 61.0.3163.100
Node    : 8.9.3
apm  2.1.2
npm  6.2.0
node 8.9.3 x64
atom 1.32.1
python 3.7.0
git 2.17.1
```

### Output From `go env`

```
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/ben/Library/Caches/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
GOPATH="/Users/ben/spell"
GOPROXY=""
GORACE=""
GOROOT="/usr/local/Cellar/go/1.11.1/libexec"
GOTMPDIR=""
GOTOOLDIR="/usr/local/Cellar/go/1.11.1/libexec/pkg/tool/darwin_amd64"
GCCGO="gccgo"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/68/c2qg0pm111l1m549k_dntx080000gn/T/go-build361782679=/tmp/go-build -gno-record-gcc-switches -fno-common"
```

### Steps to Reproduce

1. Build a struct with some types:
```
type RandomStruct struct {
	ID               int        `json:"id"`
	LastRunTime      *time.Time `json:"last_run_time"`
	DeletedAt        *time.Time `json:"deleted_at"`
}
```
2. Cmd+Click (or whatever hyperclick is bound to) will work on `LastRunTime` (taking you nowhere as expected) but will not work on `Time` which should take you to the `Time` type in the `time` package. This used to work last week.

### Expected Behavior

I expect to be able to `go to definition` on types, not just variables.

### Actual Behavior

It doesn't allow me to click at all

**Comment by bensussman** [comment]
Thanks for fixing so quickly @zmb3 When can we expect this in an official release on Atom? it looks like the build is failing on the current HEAD of master https://github.com/joefitzgerald/go-plus ?

**Comment by zmb3** [comment]
I am aiming to get a release out this weekend.  Build has been flaky lately.

**Comment by bensussman** [comment]
@zmb3 sorry to keep pestering you, but I just got an update to go-plus to version `5.9.0`, but it seems that the "inability to click on types" bug still remains.

**Comment by bogdanpetrea** [comment]
I experience the same problem as @bensussman.

**Comment by zmb3** [comment]
Just published v5.9.1 which should resolve the issue for you @bensussman @bogdanpetrea

**Comment by bogdanpetrea** [comment]
Can confirm it works. Thanks a bunch!
